src/index.d.ts

```diff
@@ -203,6 +203,25 @@ declare namespace SDKStandardComponents {
         quote: TQuotesIDPutResponse;
     }
 
+    type AuthenticationType = 'OTP' | 'QRCODE' | 'U2F'
+      
+    type AuthenticationValue = {
+        pinValue: string;
+        counter: string;
+    }
+    
+    type AuthenticationInfo = {
+        authentication: AuthenticationType;
+        authenticationValue: AuthenticationValue | string;
```

Comment:
seems odd to allow an object and a string for this

Reply:
@akapur99 string is for OTP flow, AuthenticationValue for U2F